lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -127,10 +125,9 @@ def allocate_delay
     end
 
     def allocate_delay=(val)
-      @set_params = {}
-      @set_params[:state] = val ? '' : 'no'
-      @set_params[:delay] = val ? val : ''
-      config_set('fabricpath', 'allocate_delay', @set_params)
+      state = val ? '' : 'no'
+      delay = val ? val : ''
+      config_set('fabricpath', 'allocate_delay', state: state, delay: delay)
     rescue Cisco::CliError => e
```

Comment:
Yes remove all CliError rescues that follow this same pattern,

And yes because the feature is not supported, I want to see that 3K or 9K can still run the tests gracefully